### PR TITLE
add scandir recipe

### PR DESCRIPTION
**Pull request by kain88-de**
This should work on windows too. But I never build it on windows so far. Lets see it it builds.

### CODE REVIEW

File: recipes/scandir/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - scandir
+
+about:
+  home: https://github.com/benhoyt/scandir
+  license: BSD License
```
Comment (jakirkham) [inline recipes/scandir/meta.yaml]: Please specify clauses (e.g. `BSD 3-Clause`).

File: recipes/scandir/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+{% set sha256 = "e31c0ce9a21cbb4f3fe838fedd7072bdd0b233d80203eafe46872a02f985c1ce" %}
+
+package:
+  name: scandir
+  version: {{ version }}
+
+source:
+  fn: simplejson-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/scandir/scandir-{{ version }}.tar.gz
+  # md5 is OK but sha256 is desirable.
```
Comment (jakirkham) [inline recipes/scandir/meta.yaml]: Please drop comment.

File: recipes/scandir/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
```
Comment (jakirkham) [inline recipes/scandir/meta.yaml]: Please drop comment.

File: recipes/scandir/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+{% set version = "1.2" %}
+{% set sha256 = "e31c0ce9a21cbb4f3fe838fedd7072bdd0b233d80203eafe46872a02f985c1ce" %}
+
+package:
+  name: scandir
+  version: {{ version }}
+
+source:
+  fn: simplejson-{{ version }}.tar.gz
```
Comment (jakirkham) [inline recipes/scandir/meta.yaml]: Please change `simplejson` -> `scandir`.

**Comment by kain88-de** [comment]
fixed all comments

**Comment by jakirkham** [comment]
Thanks @kain88-de.

**Comment by kain88-de** [comment]
No I have to thanks this conda-forge setup is very nice. I'll upload some more packages soon. The last two are only dependencies missing for [datreant](http://datreant.org/), that are the actual packages I want to upload.

**Comment by jakirkham** [comment]
Wouldn't be much use having a nice setup if there weren't contributors. 😄

> The last two are only dependencies missing for [datreant](http://datreant.org/), that are the actual packages I want to upload.

Interesting. Not sure I'm familiar with that package. Would like to learn though.

**Comment by jakirkham** [comment]
@benhoyt @tjguk @haypo, just letting you know we have added `scandir` to [conda-forge]( http://conda-forge.github.io/ ). This will allow users to easily install this with [`conda`]( http://conda.pydata.org/docs/ ) ([`conda install -c conda-forge scandir`]( https://anaconda.org/conda-forge/scandir )). The maintenance burden here is light as all testing releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe in [this repo]( https://github.com/conda-forge/scandir-feedstock ). If you have any interest in being added as a maintainer (to make new releases for instance), we would love to have you on board. Thanks for your work on `scandir`. 😄

**Pull request merged**